### PR TITLE
test pr

### PR DESCRIPTION
**Pull request by sarahrudy**
test pr

### CODE REVIEW

File: lib/board.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 
 class Board
-  attr_reader :cells
+  attr_reader :cells,
+              :test
```
Comment (Mstandley1985) [inline lib/board.rb:4]: Wow great fix.

Comment (sarahrudy) [reply lib/board.rb]: good catch. just pushed

**Review by Mstandley1985 — APPROVED** [review]
See ya tomorrow morning.

**Comment by Mstandley1985** [comment]
And this concludes the lesson on GitHub

**Pull request closed without merging**